Gemfile

```diff
@@ -8,6 +8,7 @@ gemspec
 # jquery-rails is used by the dummy application
 gem "jquery-rails"
 gem "mysql2"
+gem "bootstrap-datepicker-rails"
```

Comment:
Why are the other gems listed in there, but others listed in the gemspec? Is it because Hubstats is a gem/plugin, not an actual application?